examples/all-clusters-app/esp32/sdkconfig.defaults

```diff
@@ -30,7 +30,7 @@ CONFIG_ESPTOOLPY_MONITOR_BAUD=115200
 #enable BT
 CONFIG_BT_ENABLED=y
 CONFIG_BT_NIMBLE_ENABLED=y
-
+CONFIG_BT_NIMBLE_HOST_BASED_PRIVACY=y
```

Comment:
This is not required for C3. For the compilation error, you should make a conditional call to the pvcy API.

Reply:
fixed